### ISSUE TITLE
Add help text for alt text

### ISSUE THREAD

**Issue by kmranjo** [weather-gov/weather.gov#1200]
### Current state
When a user creates a Weather Story, they have the option to upload an image file to the Forecaster Image section. 
Once an image is uploaded, the Alt Text field appears, with the following help text below: 
_Short description of the image used by screen readers and displayed when the image is not loaded. This is important for accessibility._

User feedback during testing reflects that forecasters understand that alt text is required, but could use more guidance around what to include in it. 

### Acceptance criteria 
The current help text for the alt text field is replaced with: 
_Alt text should provide the same information as the image, not serve as a description of the image itself. If you have included text on your image, it should be in the alt text along with whatever additional context is needed to provide the same information to users who cannot view the image._
